op-proposer/Makefile

```diff
@@ -28,7 +28,24 @@ clean:
 test:
 	go test -v ./...
 
+abis = ../packages/contracts-bedrock/snapshots/abi
+
+bindings-upgrades:
+	bash -c ' \
+	set -euxo pipefail; \
```

Comment:
what's this?

Edit:

![image](https://github.com/babylonchain/optimism/assets/111917526/fb02eaac-ac08-4482-a875-b27af1d7ebca)